dashboard/app/models/levels/match.rb

```diff
@@ -39,4 +39,8 @@ def dsl_default
   def supports_markdown?
     true
   end
+
+  def icon
+    'fa-check-square-o'
+  end
```

Comment:
A bunch of the other level types we care about inherit from Match, and thus get the benefits of this icon.